tools/nni_cmd/launcher.py

```diff
@@ -386,6 +386,13 @@ def set_platform_config(platform, experiment_config, port, config_file_name, res
             raise Exception(ERROR_INFO % 'Rest server stopped!')
         exit(1)
 
+def auto_gen_search_space_dry_run(code_dir, command):
+    '''dry run trial code to generate search space file'''
+    print_normal('Dry run to generate search space...')
+    Popen(command, cwd=code_dir, env=dict(os.environ, NNI_GEN_SEARCH_SPACE='auto_gen_ss'), shell=True).wait()
```

Comment:
Could it be in tuner init?

Reply:
it can't, but we can use a dedicated `nnictl` command to generate search space.